ocaml/xapi/xapi_host.ml

```diff
@@ -2674,7 +2674,11 @@ let alert_if_tls_verification_was_emergency_disabled ~__context =
 let cert_distrib_atom ~__context ~host ~command =
   Cert_distrib.local_exec ~__context ~command
 
-let copy_primary_host_certs = Cert_distrib.copy_certs_to_host
+let copy_primary_host_certs ~__context ~host =
+  Xapi_pool_helpers.with_pool_operation ~__context ~op:`copy_primary_host_certs
```

Comment:
Is this called _on_ the primary host?

Reply:
yes

Reply:
it would be clearer if this `with_pool_operation` call was made in message_forward, so i'll move it there

Reply:
Yes indeed, if we put this sort of thing in message_forwarding, then it guaranteed to be primary-only.